spec/specification/dsl_spec.rb

```diff
@@ -253,6 +253,11 @@ module Pod
         @spec.module_name = 'Three20'
         @spec.attributes_hash['module_name'].should == 'Three20'
       end
+
+      it 'allows to specify a custom module map file' do
```

Comment:
From a linguist point you may be right, but I'd prefer to keep everything consistent. So feel free to go for it and open an englishification-of-all-the-specs PR. :stuck_out_tongue_winking_eye: